code/modules/mining/mine_turfs.dm

```diff
@@ -60,7 +60,7 @@
 
 /turf/closed/mineral/random
 	var/mineralSpawnChanceList = list(
-		/turf/closed/mineral/uranium = 5, /turf/closed/mineral/diamond = 1, /turf/closed/mineral/gold = 10,
+		/turf/closed/mineral/uranium = 5, /turf/closed/mineral/diamond = 1, /turf/closed/mineral/clown = 1, /turf/closed/mineral/gold = 10,
```

Comment:
we're not having bananium spawn outside of manually mapped locations, remove this.